### ISSUE TITLE
React: Remove Unnecessary Orange Dot from Image.

### ISSUE THREAD

**Issue by aaelsay3d** [TheOdinProject/curriculum#28382]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

I noticed that in one of the lessons of the [React course](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/react), specifically the [More On State lesson](https://www.theodinproject.com/lessons/react-new-more-on-state#how-state-updates) , there is an image with an unnecessary orange dot.

As someone with a bit of perfectionism, little things like this can be quite annoying! I suggest cropping the image to remove that dot.

Here is the image that I am talking about:

![An image showing some logs in Google Chrome](https://cdn.statically.io/gh/TheOdinProject/curriculum/bd3063e12816ac241f73daeffa600ca89e56c443/react/states_and_effects/more_on_state/imgs/00.png)

### Path

Ruby / Rails

### Lesson Url

https://www.theodinproject.com/lessons/react-new-more-on-state

**Comment by MaoShizhong** [comment]
That's fair enough. Never really noticed it myself but there's definitely no reason it should be there. Since you specified you didn't want to work on this, I'll open this issue up for assignment.

## Acceptance criteria

This process will require 2 PRs.

### First PR

- [ ] The image file used in the above lesson is replaced with a version that does not include the rogue orange dot.
- [ ] The replacement image file has the same name as the original.

### Second PR

Can only be done once the first PR has been approved and merged.

- [ ] The lesson file's image href is replaced with a statically link for the new image file. [Instructions for creating statically links for repo images](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum#create-statically-links)

---

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by mathdebate09** [comment]
Hey Mao, I can work these fixes up :)
